safe.py

```diff
@@ -236,13 +243,16 @@ def _check_node(node):
   <Return>
     None
   """
+  # Adjust line numbers to account for the encoding declaration header, 
+  # SeattleTestbed/repy_v2#95
```

Comment:
Minor: This one comment by itself makes it seem like the aim of the following code is to adjust for line numbers.